lib/scrabble.rb

```diff
@@ -0,0 +1,42 @@
+module Scrabble
```

Comment:
Overall it looks like there is too much whitespace in this file. Also, watch your indentation, lines 17-24 are misaligned and it makes it difficult to see where your blocks start and end.